tensorflow_addons/image/tests/filters_test.py

```diff
@@ -359,3 +361,45 @@ def test_symmetric_padding_with_3x3_filter_median(image_shape):
         constant_values=0,
         expected_plane=expected_plane,
     )
+
+
```

Comment:
Please add test cases for 4D images. With support for 'NHWC' or 'NCHW' data formats

Reply:
Done